src/fifelse.c

```diff
@@ -209,12 +209,12 @@ SEXP fcaseR(SEXP rho, SEXP args) {
             "Note that the default argument must be named explicitly, e.g., default=0"), narg - 2);
   }
   int nprotect=0, l;
-  int64_t len0=0, len1=0, len2=0;
+  int64_t n_ans=0, n_this_arg=0, n_undecided=0;
```

Comment:
`n_undecided` sounds good to indicate the count of the unresolved elements/cases. An alternative I can think of is `n_pending` (as in tracking how many elements are left to be evaluated)

Is `n_this_arg` meant to represent the length of the `thens` argument that is being processed in the current iteration? (I only see it being assigned to it at 285 below, so I'm wondering if it could be renamed specific to it)

Reply:
Thanks! Yes you're right about `n_this_arg`. I'm roughly indifferent between `n_undecided` and `n_pending`; both are way better than `len2` :)